lang/cpp/cpp.py

```diff
@@ -20,12 +34,60 @@ def cpp_standard_type(m) -> str:
     return "std::" + m[1]
 
 
+@mod.capture(rule="{user.cpp_standard_prefix} {user.cpp_standard_function}")
+def cpp_standard_function(m) -> str:
+    # Discard the prefix word and insert properly prefixed function call
+    return "std::" + m[1]
+
+
+@mod.capture(rule="{user.cpp_standard_prefix} {user.cpp_standard_constant}")
+def cpp_standard_constant(m) -> str:
+    # Discard the prefix word and insert properly prefixed function call
```

Comment:
This comment says this is inserting a properly prefixed function call even though it is for a constant